sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterImpl.java

```diff
@@ -299,6 +301,11 @@ public String agentPoolResourceGroup() {
         return innerModel().nodeResourceGroup();
     }
 
+    @Override
+    public PublicNetworkAccess publicNetworkAccess() {
+        return Objects.isNull(this.innerModel().publicNetworkAccess()) ? null : this.innerModel().publicNetworkAccess();
```

Comment:
What's difference from just `return this.innerModel().publicNetworkAccess()`?

Same on other instances.